electron/walletBackup/local/index.js

```diff
@@ -27,8 +42,9 @@ export default class BackupService {
    * @memberof BackupService
    */
   saveBackup = chainify(async ({ walletId, fileId: dir, backup }) => {
-    const filePath = `${dir}/${walletId}`
-    await writeFileAsync(filePath, backup)
+    const filePath = path.join(dir, walletId)
+    await mkdirAsync(filePath, { recursive: true })
+    await writeFileAsync(path.join(filePath, config.backup.filename), backup)
     return dir
```

Comment:
Should this really be returning `dir`? Ii looks like the caller is expecting a file id. Comments also suggest this.

Reply:
renamed backupId->locationHint